tests/test_rtf_formatter.py

```diff
@@ -26,9 +31,9 @@ def _build_message(*args, **kwargs):
     result = _escape(kwargs.get('result', ''))
 
     if string is None:
-        string = ("The expected output of '{t}'\n"
-                  "\t\tShould be '{expected}'\n"
-                  "\t\tActually outputs '{result}'\n"
+        string = ("The expected output of '{t}'\n\n"
+                  "\t\tShould be '{expected}'\n\n"
+                  "\t\tActually outputs '{result}'\n\n"
```

Comment:
Why do we have a double newline here now?

Reply:
To improve readability of the pytest message when long lines wrap in the terminal / shell (see examples at the end of reply). 

Whether those custom messages are helpful or not is debatable. The `_build_message` function was made when all tests were single line input and single line output. When testing line numbering the input and output are multi-line, as are tests for the color table. 

I considered adding a command-line option to the tests which would dump RTF-test input, expected output and output to files in a specified directory: `tox -- tests/test_rtf_formatter.py --output-dir=/tmp/rtf-tests`. That would be more useful in my opinion, but requires modifications to `conftest.py` : https://docs.pytest.org/en/7.1.x/example/simple.html?highlight=addoption#pass-different-values-to-a-test-function-depending-on-command-line-options

**Double newline:**
```
E       AssertionError: The expected output of 's = "line1\nline2"\n'
E         
E                       Should be '{\rtf1\ansi\uc0\deff0{\fonttbl{\f0\fmodern\fprq1\fcharset0;}}\n{\colortbl;\n\red255\green0\blue0;\n\red0\green255\blu
e0;\n\red0\green0\blue255;\n}\n\f0\n\dntblnsbdb\n{\cf1 1  }{\cf3 s} = {\cf2 "}{\cf2 line1}{\cf2 \\n}{\cf2 line2}{\cf2 "}\par\n}\n'
E         
E                       Actually outputs '{\rtf1\ansi\uc0\deff0{\fonttbl{\f0\fmodern\fprq1\fcharset0;}}\n{\colortbl;\n\red255\green0\blue0;\n\red0\gree
n255\blue0;\n\red0\green0\blue255;\n}\n\f0\n\dntblnsbdb\n{\cf1 1  }{\cf3 s} = {\cf2 "}{\cf2 line1}{\cf2 \\n}{\cf2 line2}{\cf2 '
E         
E               (WARNING: Partial Output of Result!)
```

** Single newline:**
```
E       AssertionError: The expected output of 's = "line1\nline2"\n'
E                       Should be '{\rtf1\ansi\uc0\deff0{\fonttbl{\f0\fmodern\fprq1\fcharset0;}}\n{\colortbl;\n\red255\green0\blue0;\n\red0\green255\blu
e0;\n\red0\green0\blue255;\n}\n\f0\n\dntblnsbdb\n{\cf1 1  }{\cf3 s} = {\cf2 "}{\cf2 line1}{\cf2 \\n}{\cf2 line2}{\cf2 "}\par\n}\n'        
E                       Actually outputs '{\rtf1\ansi\uc0\deff0{\fonttbl{\f0\fmodern\fprq1\fcharset0;}}\n{\colortbl;\n\red255\green0\blue0;\n\red0\gree
n255\blue0;\n\red0\green0\blue255;\n}\n\f0\n\dntblnsbdb\n{\cf1 1  }{\cf3 s} = {\cf2 "}{\cf2 line1}{\cf2 \\n}{\cf2 line2}{\cf2 '
E               (WARNING: Partial Output of Result!)
```

Reply:
Works for me, thanks!